packages/scope-manager/tests/fixtures.test.ts

```diff
@@ -36,7 +36,7 @@ const fixtures = glob
     };
   });
 
-const FOUR_SLASH = /^\/\/\/\/[ ]+@(\w+)[ ]*=[ ]*(.+)$/;
```

Comment:
(musing, no action requested) Wonder if someone did this as an intentional style once upon a time to make the space "more obvious" or somehting? Either way, I'm in favor the change 🤷‍♂️